### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by zarvik**
Muudetud Jaa! kirjeldus. Eemaldatud stuudio link

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -37,15 +37,16 @@
...
+                        Etenduste formaatides pakub Jaa! publikule võimalust saada osa nii lühivormi sketšidest kui pikavormi etendustest. 
+                        Jaa !mproteater on alati valmis väljakutseteks ning otsib huviga uusi vorme ja võimalusi spontaansete teatrihetkede jagamiseks!
                     </p>
 
                     <p class="lead text-center links-group">
                         <a href="http://improv.ee/trupid/improgrupp-jaa/">
                             <strong>Mängukava</strong> </a> |
-                        <a href="http://kultuur.ee/index.php?option=com_content&amp;view=article&amp;id=569&amp;Itemid=620">
-                            <strong>Impro trennid</strong>
                         </a> |
```
Comment (anroots) [inline public/index.html]: There's one extra "pipe" symbol between "Mängukava" and "Facebook" - perhaps that should be deleted as well?
selection_002

File: public/index.html [hunk 2]
```diff
@@ -37,15 +37,16 @@
 
                 <div class="content-box">
                     <p class="lead">
-                        Oleme Jaa !mproteater, spontaanne ning lõbus grupp noori amatöörnäitlejaid. Pakume publikule
-                        improviseeritud teatrietendustega rõõmu ja draamat ning õpetame huvilistele improviseerimist.
+                        Jaa !mproteater on Eesti teatrimaastikul tegutsenud aastast 2009 (enne 2016 nimega Improgrupp Jaa!). 
+                        Olles üks esimesi ja vanimaid improteatreid Eestis, on Jaa!l olnud määrav osa kohaliku improkultuuri arendamisel ja suunamisel.  
+                        Jaa! on andnud etendusi nii Eestis kui välismaal, toonud kohaliku publiku ette nii koolitajaid kui teatritruppe teistest riikidest ning korraldanud rahvusvahelist improteatrite festivali Tilt.
```
Comment (anroots) [inline public/index.html]: We could link to Tilt website.

**Comment by anroots** [comment]
Great, thanks! Looks good. Added some minor comments, let me know whether they're reasonable or should we ship to production as-is.

File: public/index.html [hunk 3]
```diff
@@ -37,15 +37,16 @@
 
                 <div class="content-box">
                     <p class="lead">
-                        Oleme Jaa !mproteater, spontaanne ning lõbus grupp noori amatöörnäitlejaid. Pakume publikule
-                        improviseeritud teatrietendustega rõõmu ja draamat ning õpetame huvilistele improviseerimist.
+                        Jaa !mproteater on Eesti teatrimaastikul tegutsenud aastast 2009 (enne 2016 nimega Improgrupp Jaa!).
```
Comment (anroots) [inline public/index.html]: On formatting: would it look prettier with some paragraph breaks and perhaps with `<p class="text-justify">` (justify alignment)?

Comment (zarvik) [inline public/index.html]: good idea!

File: public/index.html (hunk 2 above)
Comment (zarvik) [inline public/index.html]: good thinking.

**Comment by zarvik** [comment]
Fixed mentioned issues. IMHO Ready for release.

**Pull request merged**